src/Gui/Tree.cpp

```diff
@@ -2664,6 +2718,9 @@ void TreeWidget::setupText()
     this->finishEditingAction->setText(tr("Finish editing"));
     this->finishEditingAction->setStatusTip(tr("Finish editing object"));
 
+    this->selectAllChildrenAction->setText(tr("Select also all children"));

```

Comment:
This wording feels awkward to me: would "Add children to selection" work instead?

Reply:
Yes, this is better, fixed now.